server/embed/config_tracing.go

```diff
@@ -23,7 +23,7 @@ import (
 	"go.opentelemetry.io/otel/propagation"
 	"go.opentelemetry.io/otel/sdk/resource"
 	tracesdk "go.opentelemetry.io/otel/sdk/trace"
-	semconv "go.opentelemetry.io/otel/semconv/v1.4.0"
+	semconv "go.opentelemetry.io/otel/semconv/v1.17.0"
```

Comment:
It seems that the latest version is [v1.18.0](https://github.com/open-telemetry/opentelemetry-go/tree/v1.14.0/semconv/v1.18.0)? I just want to know if you intentionally selected 1.17.0.

Reply:
This is intentional — v1.17.0 is the latest version used in etcd’s dependencies.

Reply:
The latest version included in otel is v1.18.0, the latest version actually used is v1.17.0: https://github.com/search?q=repo%3Aopen-telemetry%2Fopentelemetry-go+semconv%2Fv+-path%3Asemconv&type=code

Reply:
OK, thx for the clarification.